src/electionguard/guardian.py

```diff
@@ -154,6 +154,13 @@ def __init__(
         sequence_order: int,
         number_of_guardians: int,
         quorum: int,
+        election_keys: ElectionKeyPair = None,
+        backups_to_share: Dict[GuardianId, ElectionPartialKeyBackup] = None,
```

Comment:
We should not have to preload the backups_to_share OR the election_partial_key_verifications. There is no use case for that.

Reply:
It feels unbalanced to export data in `export_private_data` that isn't imported in `from_private_record`.  How would you feel if instead I were to remove `backups_to_share` `election_partial_key_verifications` from both?  It's currently only used by `TestEndToEndElection`, and I don't think it would break anything there to remove them there.

Reply:
That's fine. That's a straightforward solution.